### ISSUE TITLE
Extension set to WebP no longer works

### ISSUE THREAD

**Issue by pcfreak30** [rosell-dk/webp-express#304]
@rosell-dk 

Recent commit 21335d29fdd7ab8c6747e02b5edface624c3f9c4 line 84-89 is causing the desination to always append by clobbering the `$desination` variable

```
 $destination = preg_replace('/\\.(jpe?g|png)$/', '', $source) . '.webp';
                    // TODO: make this check work with symlinks
                    //$destination = SanityCheck::absPathIsInDocRoot($source . '.webp');
                    $destination = $source . '.webp';
```

It looks like this was a mistake? Would appreciate a commit fixing this as it threw me for a loop today on a deploy.

Thanks :)

**Comment by rosell-dk** [comment]
Hi Derrick,

Sorry for the delay. I have been off for vacation.

Yes, you are right. This bug must have been some blunder. I have fixed it in #311. I expect to release in a couple of days (I'm in the process of digging through the pile of feedback received while on vacation, and there might be other bugs there)

**Comment by rosell-dk** [comment]
The fix has just been released (0.14.22)
